src/vm/errors.nim

```diff
@@ -218,6 +218,10 @@ proc SyntaxError_EmptyLiteral*(lineno: int, context: string) =
 proc AssertionError_AssertionFailed*(context: string) =
     panic AssertionError,
           context
+          
+proc AssertionError_AssertionFailed*(context: string, message: string) =
+    panic AssertionError, 
+          "\"" & message & "\": " & context
```

Comment:
```suggestion
          message & ":;" & 
          "for: " & context
```

`;`s are a hack to denote *line-change* within an error body. I think - just think - it will hopefully look cleaner and more uniform this way... Also, I don't really think we need the double quotes, IMHO. 😉